test/unit/test_job_pytest.py

```diff
@@ -143,11 +153,15 @@ def test_recover_last_log_name(tmpdir, test_with_logfiles, file_timestamp_greate
     },
     {'notify_on': ['COMPLETED']}
 )])
-def test_update_parameters_attributes(autosubmit_config, experiment_data, attributes_to_check):
+def test_update_parameters_attributes(
+        autosubmit_config,
+        experiment_data,
+        attributes_to_check):
     job = create_job_and_update_parameters(autosubmit_config, experiment_data)
     for attr in attributes_to_check:
         assert hasattr(job, attr)
-        assert getattr(job, attr) == attributes_to_check[attr]
+        assert getattr(job, attr) == attributes_to_check[attr] if \
+                type(attributes_to_check[attr]) is not list else attributes_to_check[attr][0]
```

Comment:
Too many spaces :grimacing: 

And missing one blank space after the function:

![image](https://github.com/user-attachments/assets/09131aa1-fb12-4379-a5c9-09daeaa4d770)